assets/queries/dockerfile/copy_instead_of_add/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "Copy_instead_of_add",
+  "queryName": "Copy Instead of Add",
```

Comment:
The query name should identify the problem, in this case, that the Dockerfile is using ADD command instead of COPY